packages/disploy/src/structs/UserInteraction.ts

```diff
@@ -36,7 +36,7 @@ export class UserInteraction extends BaseInteraction {
 	 */
 	public targetUser!: User | null;
 
-	public constructor(app: App, public raw: APIUserApplicationCommandInteraction) {
+	public constructor(app: App, public override raw: APIUserApplicationCommandInteraction) {
```

Comment:
While this would probably be a good idea to let the user access the raw objects; it is out of scope for this PR.